CODEOWNERS

```diff
@@ -17,6 +17,9 @@
 /contrib/**/*appsec*.go         @DataDog/asm-go
 /.github/workflows/appsec.yml   @DataDog/asm-go
 
+# datastreams
+/datastreams                    @Datadog/data-streams-monitoring
```

Comment:
We need to fix this 
![image](https://github.com/DataDog/dd-trace-go/assets/38987709/6a632c8f-8ecd-4398-bc23-672834b81d6c)